README.md

```diff
@@ -227,7 +227,7 @@ spec:
   type: NodePort
 
 ---
-apiVersion: vanus.linkall.com/v1alpha1
+apiVersion: vanus.vanus.ai/v1alpha1
```

Comment:
is it better that remove prefix of 'vanus'?

Reply:
vanus.vanus.ai is {product name}.{domain}，like rocketmq.apache.org